app/views/course/quiz_questions/_card_header.html.erb

```diff
@@ -4,13 +4,13 @@
   data-target="#collapse-<%= quiz_question.id %>"
   aria-controls="collapse-<%= quiz_question.id %>">
 
-  <div class="btn btn-light bg-transparent border-0">
+  <div class="btn btn-light bg-transparent border-0 text-left">
```

Comment:
I found the issue. It's a side effect of me forgetting to remove the btn classes after a change. There's bootstrap override, that I often do, that sets `btn` to `text-transform: uppercase`. Leaving just `text-left` here should be enough.

Reply:
Are you fixing the bootstrap override or should I go find it and remove it?

Reply:
The bootstrap override is good. Makes the button look better in general. The issue here is that this is not a button and shouldn't have this class. So yeah. We're removing it here.

Reply:
I found where you did the override so let me know if I should remove it. I think if I do remove it, it will also un-capitalize the navigation items. Not sure if we want that or not.

Reply:
Got it!